README.md

```diff
@@ -11,16 +11,17 @@ Debian
-| `rsync_backup_short_flags`  | _see [defaults/main.yml](defaults/main.yml)_ | A list of single-char flags to set when executing rsync, like `a` or `h`                                                     |
-| `rsync_backup_excludes`     | _see [defaults/main.yml](defaults/main.yml)_ | A list of paths to exclude from backing up. Paths are relative to the source and target paths if they don't start with a `/` |
-| `rsync_backup_frequency`    | `daily`                                      | systemd execution frequency of the timer                                                                                     |
+| Variable                          | Default                                      | Description                                                                                                                  |
+|:----------------------------------|:---------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------|
+| `rsync_backup_source_paths`       | `['/', '/boot']`                             | List of source paths that needs to be backed up                                                                              |
+| `rsync_backup_target_path`        | `/mnt/backup`                                | Destination path to backup to                                                                                                |
+| `rsync_backup_mount_pre`          | `True`                                       | Whether to mount the target path before rsyncing (implies that there has to be a entry for it in `/etc/fstab`)               |
+| `rsync_backup_umount_post`        | `True`                                       | Whether to unmount the target path after rsyncing (implies that there has to be a entry for it in `/etc/fstab`)              |
+| `rsync_backup_flags`              | _see [defaults/main.yml](defaults/main.yml)_ | A list of flags to set when executing rsync, like `delete-during` or `stats`                                                 |
+| `rsync_backup_short_flags`        | _see [defaults/main.yml](defaults/main.yml)_ | A list of single-char flags to set when executing rsync, like `a` or `h`                                                     |
+| `rsync_backup_excludes`           | _see [defaults/main.yml](defaults/main.yml)_ | A list of paths to exclude from backing up. Paths are relative to the source and target paths if they don't start with a `/` |
+| `rsync_backup_frequency`          | `daily`                                      | systemd execution frequency of the timer                                                                                     |
+| `rsync_backup_allowed_exit_codes` | `[0]`                                        | List of allowed return codes for `rsync`.                                                                                    |
```

Comment:
Please use the same rsync_backup_allowed_return_codes, exit_codes is a bit confusing.